database/migrations/2019_09_28_171753_config_fix.php

```diff
@@ -0,0 +1,577 @@
+				'confidentiality' => '4',
+			],
+			[
+				'key' => 'password',
+				'value' => '',
+				'cat' => 'Admin',
+				'type_range' => STRING_REQ,
+				'confidentiality' => '4',
+			],
+			[
+				'key' => 'check_for_updates',
+				'value' => '0',
+				'cat' => 'Admin',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'sorting_Photos_col',
+				'value' => 'title',
+				'cat' => 'Gallery',
+				'type_range' => 'id|takestamp|title|description|public|star|type',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'sorting_Photos_order',
+				'value' => 'ASC',
+				'cat' => 'Gallery',
+				'type_range' => 'ASC|DESC',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'sorting_Albums_col',
+				'value' => 'max_takestamp',
+				'cat' => 'Gallery',
+				'type_range' => 'id|title|description|public|max_takestamp|min_takestamp',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'sorting_Albums_order',
+				'value' => 'ASC',
+				'cat' => 'Gallery',
+				'type_range' => 'ASC|DESC',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'imagick',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'dropbox_key',
+				'value' => '',
+				'cat' => 'Admin',
+				'type_range' => STRING,
+				'confidentiality' => '3',
+			],
+			[
+				'key' => 'skip_duplicates',
+				'value' => '0',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'lang',
+				'value' => 'en',
+				'cat' => 'Gallery',
+				'type_range' => DISABLED,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'layout',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => TERNARY,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'image_overlay',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'default_license',
+				'value' => 'none',
+				'cat' => 'Gallery',
+				'type_range' => STRING_REQ,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_max_width',
+				'value' => '0',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_max_height',
+				'value' => '360',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_max_width',
+				'value' => '1920',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_max_height',
+				'value' => '1080',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'full_photo',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'delete_imported',
+				'value' => '1',
```

Comment:
I recently changed this to `0` since I felt that defaulting to a destructive behavior is not user-friendly. Though, given that it's configurable, it's not a big deal.

Reply:
Agreed. Especially as there's a checkbox, I'd default to non-destructive.

Reply:
I am not changing the value if there is one already set.

Reply:
No, but @kamil4 changed the default in #336 (93448cc426794a3f465725b172aa3f04cf28d31b) and this would revert it.